plugin/template/README.md

```diff
@@ -96,6 +96,37 @@ The most simplistic template is
 2. All queries will be answered (no `fallthrough`)
 3. The answer is always NXDOMAIN
 
+### Resolving SOA with dynamic HOSTNAME and static NS entries
+
+Return a start of authority record with a dynamic hostname:
+
+~~~ corefile
+engineering.coredns.io. {
+    template IN SOA {
+      match ^engineering\.coredns\.io\.$
+      answer "{{ .Zone }} 3600 IN SOA {$HOSTNAME}. engineering.coredns.io. (1 60 120 86400 3600)"
```

Comment:
I don't think the SOA field values are correct here.

First field after record type should be the primary master namserver.  e.g. `ns1.engineering.coredns.io.`
Second field after record type should be an encoded admin email address. e.g. `admin.engineering.coredns.io.`

```suggestion
      answer "{{ .Zone }} 3600 IN SOA ns1.{{ .Zone }} admin.{{ .Zone }} (1 60 120 86400 3600)"
```

This would be the same for all returned SOA records.  But then, the dynamic hostname isn't used anywhere, so. I'm confused by the example. I'm probably missing something.